__tests__/topics.test.js

```diff
@@ -0,0 +1,52 @@
+const app = require('../serverFiles/app.js');
+const testData = require('../db/data/test-data/index.js');
+const db = require('../db/connection.js');
+const seed = require('../db/seeds/seed.js');
+const request = require('supertest');
+
+beforeEach(() => {
+  return seed(testData);
+});
+
+afterAll(() => {
+  return db.end();
+});
+
+describe('CORE: GET /api/topics', () => {
+  test('Should return an array of topic objects', () => {
+    return request(app)
+      .get('/api/topics')
+      .expect(200)
+      .then(({ body }) => {
+        expect(Array.isArray(body.allTopics)).toBe(true);
+      });
+  });
+  test('Should return ALL topics in the array', () => {
+    return request(app)
+      .get('/api/topics')
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.allTopics).toHaveLength(3);
+      });
+  });
+  test('Returned topic objects should have the properties slug and description', () => {
+    return request(app)
+      .get('/api/topics')
+      .expect(200)
+      .then(({ body }) => {
+        const allTopics = body.allTopics;
+        allTopics.forEach((topic) => {
+          expect(topic).toHaveProperty('slug', expect.any(String));
+          expect(topic).toHaveProperty('description', expect.any(String));
+        });
+      });
+  });
+  test('404: Error sent if endpoint does not exist', () => {
+    return request(app)
+      .get('/')
```

Comment:
You have passed this test - but if you changed your invalid endpoint - would it still pass? Eg `/api/invalidpath`
We need a solution that is triggered by any invalid path